cmake/configs/SurelogConfig.cmake.in

```diff
@@ -0,0 +1,12 @@
+set(UHDM_VERSION "@PROJECT_VERSION_MAJOR@.@PROJECT_VERSION_MINOR@")
+
+@PACKAGE_INIT@
+
+include(${CMAKE_CURRENT_LIST_DIR}/SurelogTargets.cmake)
+
+set_and_check(SURELOG_BIN_DIR @CMAKE_INSTALL_PREFIX@/bin)
+set_and_check(SURELOG_INCLUDE_DIR @CMAKE_INSTALL_PREFIX@/include/Surelog)
+set_and_check(SURELOG_INCLUDE_DIRS @CMAKE_INSTALL_PREFIX@/include/Surelog)
+set_and_check(SURELOG_LIB_DIR @CMAKE_INSTALL_PREFIX@/lib/surelog)
```

Comment:
Do do the proper lib64 situation that we also had to fix in pkg-config, wouldn't this be `@CMAKE_INSTALL_FULL_LIBDIR@/surelog` ?